api/location/schema.py

```diff
@@ -72,8 +75,9 @@ def mutate(self, info, location_id, **kwargs):
             validate_url(**kwargs)
         if "abbreviation" in kwargs:  # noqa
             validate_empty_fields(**kwargs)
-        result = location.filter(LocationModel.name == kwargs.get('name'))
-        if result.count() > 0:
+        active_locations = result.filter(
+            LocationModel.name == kwargs.get('name'))
+        if active_locations.count() > 0:
```

Comment:
Why is counting necessary at this point? Could it be possible to check active locations without counting?

Reply:
@Bonifase ,I have implemented the changes.